### ISSUE TITLE
New Field "junction table" dropdown

### ISSUE THREAD

**Issue by laszlokorte** [directus/directus#1170]
## Version Info
  - Directus version: d2128abec

## Expected Behavior

When creating a new many-to-many field you need to select a "Junction Table". After selecting the a table from the dropdown the dropdown's value does not change. After selecting the same table twice the dropdown's value finally changes.

## Actual Behavior

The dropdown's value should change on the first attempt.

## Steps to Reproduce
 1. Create a new field
 2. Chose the "Many to Many" UI
 3. select a junction table
 3b. notice that the dropdown's value does not change to the table you just chose.
 4. select the same junction table again
 4b. now the value changes

## issue

I guess this is caused by rerendering the form whenever the the junction table value changes in order to adjust the column select boxes below.

**Comment by benhaynes** [comment]
Good catch! Field creation (especially relational) is something we're trying to refine and polish. This testing is invaluable – and I think @WellingGuzman is about to push a hotfix for this one.

Thanks again, @laszlokorte !

**Comment by WellingGuzman** [comment]
@laszlokorte I fixed this issue with this commit https://github.com/directus/directus/commit/b863e1029968a1aba26f00bf45a9e0b7712f0b71.

Let us know if all good in your side.

Thanks for reporting this one.
